### PR TITLE
Implement the at-use-no-unnamespaced rule

### PR DESCRIPTION
**Pull request by stof**
Closes https://github.com/stylelint-scss/stylelint-scss/issues/565

### CODE REVIEW

**Comment by stof** [comment]
My implementation is incomplete. It won't catch configured modules, as the configuration comes after the namespace in the rule.

**Comment by stof** [comment]
And here is the complete implementation

File: src/rules/at-use-no-unnamespaced/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("at-use-no-unnamespaced");
+
+export const messages = utils.ruleMessages(ruleName, {
+  expected: "Expected @use to use a namespace"
```
Comment (kristerkari) [inline src/rules/at-use-no-unnamespaced/index.js:7]: I went and checked from the stylelint repo, and most of the `-no-` rules seem to use the `rejected: "Unexpected ..."` message format, should this rule also use it?

Comment (stof) [reply src/rules/at-use-no-unnamespaced/index.js]: I updated it

**Comment by kristerkari** [comment]
This was released in 4.1.0:
https://github.com/stylelint-scss/stylelint-scss/releases/tag/v4.1.0

Thank you @stof !

**Pull request merged**